recipes/molearn/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "molearn" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Degiacomi-Lab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: ed43bbe23ec5be2cd25a8e5970fa255f67404ec62bdb269ba3489d97be3bcb8b
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [ win ]
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - pip
+  run:
+    - python
+    - numpy
+    - pytorch
+    - biobox
+test:
+  imports:
+    - {{ name }}
```

Please don't use a variable here as that confuses our bots. Additionally, it is good to have a pip check to make sure that the dependencies are correct.
```suggestion
    - molearn
  commands:
    - pip check
  requires
    - pip
```

---

I can't find anyone else who has heard of variables confusing bots

---

It is also non-standard. Please change it.

---

Seems like it's just a whim but I will change it